tests/integration/telemetry/stackdriver/vm/testdata/trace.prototext.tmpl

```diff
@@ -15,6 +15,7 @@ spans:{
     labels:{key:"response_flags"  value:"-"}
     labels:{key:"span"  value:"server.{{ .EchoNamespace }}.svc.cluster.local:8090/*"}
     labels:{key:"upstream_cluster"  value:"inbound|8090||"}
+    labels:{key:"upstream_cluster.name"  value:"inbound|8090||"}
```

Comment:
curious: why the new trace info with `.name` to match `upstream_cluster`? is the new Envoy SHA pushing out (duplicative) cluster info ?

Reply:
Yeah: https://github.com/envoyproxy/envoy/commit/5551e0a3e442cca24d76eeea76e8b9dc5f894303

Reply:
looks like: https://github.com/envoyproxy/envoy/pull/15139.

@bianpengyuan can we use this (potentially) in combo with the metadata stuff to provide human-readable names here (a la `istio.canonical_service` and `istio.canonical_revision`) ?

Reply:
I do like using this new field to carry other readable useful information, but I am not sure if canonical service is a good fit here. I think it would be better to add it with a separated label. How about just putting destination service here? We actually don't have that in the span right now.

Reply:
destination service should be ~= to `http.url`, right?  but yes, anything would be better than repeating the same cluster name. whatever our choice, it will likely show up in a few different telemetry places, so we should pick wisely.

Reply:
Actually.. I am worried that if `alt_stat_name` will also change envoy native stats.. so this will need more consideration. I created https://github.com/istio/istio/issues/31244 to track this.